src/routes/tokens.rs

```diff
@@ -6,16 +6,8 @@ use crate::errors::UserError;
 use crate::utils;
 
 pub fn get_tokens() -> Result<HttpResponse, UserError> {
-    let mut db = Database::new()
-        .map_err(|e| {
-            error!(utils::LOGGER, "{}", e);
-            UserError::Internal
-        })?;
-    let tokens = db.get_tokens()
-                   .map_err(|e| {
-                       error!(utils::LOGGER, "{}", e);
-                       UserError::Internal
-                   })?;
+    let mut db = Database::new()?;
+    let tokens = db.get_tokens()?;
     let tokens_json = serde_json::to_value(tokens)
```

Comment:
We could even do the Serde errors the same way as the postgres one. But I am not sure if that's a just assumption. (Although I can't think of anything except an internal error for it)